ballot_api/settings/heroku.py

```diff
@@ -0,0 +1,20 @@
+# flake8: noqa
+
+import os
+
+import dj_database_url
+
+from .common import *
+
+DEBUG = False
+
+ALLOWED_HOSTS = [
+    'caciviclab-ballot-api.herokuapp.com',
+]
+
+DATABASES['default'] = dj_database_url.config()
+
+SECRET_KEY = os.environ.get('SECRET_KEY', None)
```

Comment:
I made some comments on this in the first PR that basically boil down to this: it must be set. The app won't start. So we don't really need to throw an exception because the app will throw one for us.